lib/data/config/config.dart

```diff
@@ -2,7 +2,7 @@ class Environment {
   /// URL that is common to most outgoing requests.
   static const String baseUrl = String.fromEnvironment(
     'BASE_URL',
-    defaultValue: 'https://codephile.mdg.iitr.ac.in/v1/',
+    defaultValue: 'https://codephile-test.herokuapp.com/v1/',
```

Comment:
I assume you started a test server to test the login functionality? When I try to log in, it shows `incorrect credentials.`

Reply:
Yeah. This is hitting the debug server, and there's no facility to sign up yet. ('Cause there's no sign up screens yet.)